frontend/src/components/MultiSelection.tsx

```diff
@@ -11,9 +11,10 @@ import { MenuItemStatus } from 'pages/groupSettings/groupTypes';
 type MultiSelectionProps = {
   value: MenuItemStatus[];
   setValue: (itemSelection: MenuItemStatus[]) => void;
+  bodyText: string;
```

Comment:
nit pick. `bodyText` can be confusing as MultiSelection does not render a body text at all. When reusing the component, someone might think "whatever i put in body text will be used a visible label". I would rename it to something like ariaLabel or allow users to pass `...props` into the `<Select {...props}>` component.

Reply:
I agree with Gage here -- I think the simplest solution would be to just make the prop `ariaLabel` instead, as anything prefixed with aria is going to be clear to be a11y support.

Reply:
Thanks @Gkrumbach07 @andrewballantyne, updated